src/tests/unit/onbase-dao.test.js

```diff
@@ -0,0 +1,243 @@
+import 'axios';
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import sinon from 'sinon';
+import proxyquire from 'proxyquire';
+import 'config';
+import setCookie from 'set-cookie-parser';
+import { createConfigStub } from './test-helpers';
+
+chai.should();
+chai.use(chaiAsPromised);
+
+describe('Test emails-dao', () => {
+  const docType1 = {
+    id: '24680',
+    name: 'TestDocType',
+    typeId: '924680',
+  };
+  const fakeData = {
+    data: {
+      access_token: 'fake token',
+      keywordGuid: 'fake guid',
+      keywordCollection: 'fake keyword collection',
+      id: 'fake id',
+      items: [docType1],
+    },
+  };
+
+  const keyword1 = {
+    name: 'Keyword1',
+    typeId: '924680',
+    keywordTypeId: '924680',
+  };
+  const keyword2 = {
+    name: 'Keyword2',
+    typeId: '9123456',
+    keywordTypeId: '9123456',
+  };
+  const keywordType1 = {
+    id: '924680',
+    name: 'KeywordType1',
+  };
+  const keywordType2 = {
+    id: '9123456',
+    name: 'KeywordType2',
+  };
+  const fakeKeywordsTypesData = {
+    data: {
+      access_token: 'fake token',
+      keywordGuid: 'fake guid',
+      keywordCollection: 'fake keyword collection',
+      id: 'fake id',
+      items: [keywordType1, keywordType2],
+    },
+  };
+
+  let stubAxios;
+  let onbaseDao;
+  let stubAxios2;
+  let onbaseDao2;
+
+  beforeEach(() => {
+    createConfigStub();
+    stubAxios = sinon.stub().callsFake(
+      () => fakeData,
+    );
```

Comment:
What's the purpose of breaking lines here? Again, please ensure all of your code follows our coding conversion and follows the style guide. There are lots of styling issues among your codes; I'm not going to reply to them all individually.